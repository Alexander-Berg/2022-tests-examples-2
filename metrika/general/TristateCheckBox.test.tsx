// import * as React from 'react';
// import { ReactElementsList } from 'typings/react';
// import { jestSnapshotShallowTest } from 'testing/jest-utils';
// import TristateCheckBox from './TristateCheckBox';

// const componentsForSnapshotRenderTest: ReactElementsList = {
//     'with size s and checked indeterminate': (
//         <TristateCheckBox checked="indeterminate" theme="normal" size="s">
//             Test
//         </TristateCheckBox>
//     ),
//     'with size m and checked indeterminate': (
//         <TristateCheckBox checked="indeterminate" theme="normal" size="m">
//             Test
//         </TristateCheckBox>
//     ),
//     'with size n and checked indeterminate': (
//         <TristateCheckBox checked="indeterminate" theme="normal" size="n">
//             Test
//         </TristateCheckBox>
//     ),
//     'with size n and checked yes': (
//         <TristateCheckBox checked="yes" theme="normal" size="n">
//             Test
//         </TristateCheckBox>
//     ),
//     'with size n and checked no': (
//         <TristateCheckBox checked="no" theme="normal" size="n">
//             Test
//         </TristateCheckBox>
//     ),
// };

// describe('TristateCheckBox', () => {
//     describe('renders', () => {
//         jestSnapshotShallowTest(componentsForSnapshotRenderTest);
//     });
// });
