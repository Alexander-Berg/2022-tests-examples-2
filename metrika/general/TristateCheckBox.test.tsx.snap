// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TristateCheckBox renders with size m and checked indeterminate 1`] = `
<span
  className="TristateCheckBox TristateCheckBox_checked_indeterminate"
>
  <CheckBox
    checked={true}
    size="m"
    theme="normal"
  >
    Test
  </CheckBox>
</span>
`;

exports[`TristateCheckBox renders with size n and checked indeterminate 1`] = `
<span
  className="TristateCheckBox TristateCheckBox_checked_indeterminate"
>
  <CheckBox
    checked={true}
    size="n"
    theme="normal"
  >
    Test
  </CheckBox>
</span>
`;

exports[`TristateCheckBox renders with size n and checked no 1`] = `
<span
  className="TristateCheckBox TristateCheckBox_checked_no"
>
  <CheckBox
    checked={false}
    size="n"
    theme="normal"
  >
    Test
  </CheckBox>
</span>
`;

exports[`TristateCheckBox renders with size n and checked yes 1`] = `
<span
  className="TristateCheckBox TristateCheckBox_checked_yes"
>
  <CheckBox
    checked={true}
    size="n"
    theme="normal"
  >
    Test
  </CheckBox>
</span>
`;

exports[`TristateCheckBox renders with size s and checked indeterminate 1`] = `
<span
  className="TristateCheckBox TristateCheckBox_checked_indeterminate"
>
  <CheckBox
    checked={true}
    size="s"
    theme="normal"
  >
    Test
  </CheckBox>
</span>
`;
